source/2019-05-25-the-ember-times-issue-100.md

```diff
@@ -93,13 +94,13 @@ Please thank [@Alonski](https://github.com/Alonski), [@maxwondercorn](https://gi
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
-  
+## [Codemod for ES6 Class Syntax Including Decorators🤖](https://github.com/ember-codemods/ember-es6-class-codemod)
+Excited about the chance to try out the ✨**native decorator support**✨ announced as part of the [release of Ember.js 3.10](https://blog.emberjs.com/2019/05/21/ember-3-10-released.html)? The good news is there's a [codemod](https://github.com/ember-codemods/ember-es6-class-codemod) to help you out with that!
+
+Note that this part of the codemod is a configurable option in the transforms. If set to true, it will transform an object's properties to decorators wherever required. Also note that, unless disabled, this codemod will by default handle transforming object properties to class fields.
```

Comment:
The "this part" in the first sentence confused me a little. I believe you were talking about the option `--decorators` without naming it to give a high-level overview?

Reply:
Hey @ijlee2! The `as part` was referring to the `native decorator support` announced in the 3.10 release. The codemod helps folks who want to use decorators, but is not actually part of the 3.10 release. Do you think it needs clarification?

Reply:
Ah, I see. I didn't know that this codemod had existed before; it now also supports decorators.

I think it's all right to leave as is. I didn't make the connection to the "native decorator support" in the first sentence (low on coffee 😄).

Thanks for clarifying!